docs/API.md

```diff
@@ -1029,6 +1029,33 @@ minioClient.deleteBucketNotification('my-bucketname', function(e) {
 })
 ```
 
+<a name="getBucketPolicy"></a>
+### getBucketPolicy(bucketName, objectPrefix, callback)
+
+Get the bucket policy associated with the specified bucket. If `objectPrefix`
+is not empty, the bucket policy will be filtered based on object permissions
+as well.
+
+__Parameters__
+
+
+| Param  |  Type | Description  |
+|---|---|---|
+| `bucketName`  | _string_  | Name of the bucket |
+| `objectPrefix` | _string_ | Prefix of objects in the bucket with which to filter permissions off of. Use `''` for entire bucket. |
+| `callback(err, policy)`  | _function_  | Callback function is called with non `null` err value in case of error. `policy` will be the string representation of the bucket policy (`none`, `readonly`, `writeonly`, or `readwrite`). |
```

Comment:
Those constants aren't actually exported — they're just for internal use. I figured it'd be easier to use strings for the arguments as it's more readable, do you think so?